Wikipedia/Code/WMFImageController.swift

```diff
@@ -454,6 +454,7 @@ extension WMFImageController {
             }
 
             let success = { (didCache: Bool) in
+                self.imageManager.wmf_removeImageForURL(imageURL, fromDisk: false)
```

Comment:
Does this mean if we've successfully cached an image, we're then going to remove it from the cache?